ansible/roles/sshd/tasks/runtime.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Disallow SSH password authentication
+  lineinfile:
+    dest: /etc/ssh/sshd_config
```

Comment:
This doesn't work on RL9 - there's a /etc/ssh/sshd_config.d/50-cloud-init.conf which wins.

Reply:
darn it - a drop in config file does sound nicer